### ISSUE TITLE
Julia v1.0.0 used for Micro-Benchmarks?

### ISSUE THREAD

**Issue by MagicMuscleMan** [JuliaLang/www.julialang.org#1974]
**Is your feature request related to a problem? Please describe.**
On the [Julia Mcro-Benchmarks page](https://julialang.org/benchmarks/) it is stated, that the benchmarks show Julia v1.0.0 results. However, in the [Microbenchmark repository](https://github.com/JuliaLang/Microbenchmarks/blob/6d66e4c35241dad6eb15d5389f55f000ddfad770/.github/workflows/benchmarks.yml#L30) currently Julia v1.9.3 seems to be used.

**Describe the solution you'd like**
Either the version numbers on the benchmark page should be updated (ideally in a way to always represent the correct versions in the future) or the benchmark should be repeated with a more up-to-date Julia version.

**Comment by ViralBShah** [comment]
Maybe someone can submit a PR to the benchmarks file updating everything and seeing how it all works out: https://github.com/JuliaLang/Microbenchmarks/blob/master/.github/workflows/benchmarks.yml

**Comment by MagicMuscleMan** [comment]
[PR](https://github.com/JuliaLang/Microbenchmarks/pull/82) created. Note, that I don't know whether anything of this will work.

Even if it should work, there is still the Julia Micro-Benchmark page stating that Julia v1.0.0 is used and I do not know whether the text is incorrect or whether the figure never has been updated since v1.0.0.

**Comment by ViralBShah** [comment]
That page has not been updated since 1.0.

**Comment by MagicMuscleMan** [comment]
PR seems to build without error after updating some stuff and deactivating Go. Could you please go through it and work towards approving it? Without the PR this is in a non-useful state if I understand this correctly.

If there is someone with Go knowledge, they can reactivate the Go benchmarks later.

If the benchmark really works (I think I saw some numbers where the whole result might have been calculated during compilation), could you then also update the web page to show 1.11 numbers? As long as we do not recommend Julia 1.0 to the user, we should not show the 1.0 numbers as the official Julia performance values.

**Comment by ViralBShah** [comment]
Used the JuliaCon hackathon to clean up the repo and generated a new chart. PR: https://github.com/JuliaLang/Microbenchmarks.jl/pull/85

![Image](https://github.com/user-attachments/assets/d4ca6186-302a-4aed-8845-12209ebf9f27)

**Comment by linwaytin** [comment]
Awesome! What is the Julia version for this plot?

**Comment by ViralBShah** [comment]
1.11.

**Comment by ViralBShah** [comment]
Now the benchmarks are auto-generated in Github Actions at https://julialang.github.io/Microbenchmarks.jl/dev/. Once we bring that to parity to include a few of the missing ones (matlab, octave, go), we can update the page on julialang.org.

**Comment by linwaytin** [comment]
Awesome! Thanks for this work. 

In addition to updating the plot on `julialang.org`, I would suggest adding a link to [the auto-generated one](https://julialang.github.io/Microbenchmarks.jl/dev/) on the same [webpage](https://julialang.org/benchmarks/). 

I believe many people would like to see the newest result.

**Comment by ViralBShah** [comment]
Yes, the Microbenchmarks.jl auto-generated page could use some love on making it comprehensive with some more docs about the benchmarks.

Adding those other benchmarks should be straightforward too, since there are github actions available for all of those.

Once it is a little nicer looking, we can add the link as well. Contributions welcome!

**Comment by linwaytin** [comment]
In addition to matlab/octave and maybe go, I think we can consider adding `cython`. From the list in [Archlinux's repo](https://archlinux.org/packages/extra/x86_64/cython/), `cython` is used in many projects. Given that many python users might consider speeding up their code with `cython`, it is useful to should add it in the benchmark.

I will check it more to see if I can contribute.

**Comment by ViralBShah** [comment]
I am not sure what value cython adds, given we already have C in the benchmarks. What is more relevant is making sure the benchmarks are all roughly comparable to each other and to improve the quality of the suite.

**Comment by linwaytin** [comment]
I want to see if there is difference in performance, but yeah, I think you are right, we probably cannot see meaningful difference given the benchmark functions are pretty simple.

**Comment by ViralBShah** [comment]
In https://github.com/JuliaLang/www.julialang.org/pull/2367, I point to the new site where the benchmarks are run automatically. The big thing is that it doesn't have Matlab and Mathematica. If someone can get those running in CI on Microbenchmarks.jl, we can replace the benchmarks page completely with new versions. But until then, the best we can do is point to the new site.

**Comment by linwaytin** [comment]
I think this is good for now. I did a quick check for running in CI using Matlab/Mathematica. It seems it is possible, but a little involved.
